source/Audio.cpp

```diff
@@ -255,6 +286,90 @@ void Audio::Update(const Point &listenerPosition)
 
 
 
+void Audio::UpdateMusic(PlayerInfo &player, Track::GameState state)
+{
+	if(!isInitialized)
+		return;
+
+	// Music defined by planet
+	if(state == Track::GameState::LANDED)
+	{
+		if(!player.GetPlanet()->MusicName().empty())
+		{
+			PlayMusic(player.GetPlanet()->MusicName());
+			return;
+		}
+	}
+	else
+	{
+		if(player.GetSystem()->MusicName().empty())
```

Comment:
This could be related as to why music may not be playing but there are other issues too.

Reply:
Music interrupting while jumping across systems is likely a separate bug.

Reply:
Fixed